apps/services/user-notification/src/app/modules/notifications/notification.model.ts

```diff
@@ -0,0 +1,88 @@
+import {
+  Table,
+  Column,
+  Model,
+  DataType,
+  AutoIncrement,
+  PrimaryKey,
+  CreatedAt,
+  UpdatedAt,
+  Index,
+} from 'sequelize-typescript'
+
+interface ArgItem {
+  key: string
+  value: string
+}
+
+enum NotificationStatus {
+  READ = 'read',
+  UNREAD = 'unread',
+}
+@Table({
+  tableName: 'user_notification', // Explicitly setting the table name
+})
+@Table
+export class Notification extends Model<Notification> {
```

Comment:
This is causing you the any trouble when creating. We should use the `InferAttributes` and `InferCreationAttributes` + `CreationOptional` described in [Sequelize docs](https://sequelize.org/docs/v6/other-topics/typescript/#getting-a-models-attributes)

```suggestion
export class Notification extends Model<InferAttritbutes<Notification>, InferCreationAttribtues>> {

// then you wrap the type of attributes that are optional on creation like this
id!: CreationOptional<number>
created: CreationOptional<Date>
...
```